code/go/0chain.net/smartcontract/minersc/sharder.go

```diff
@@ -349,3 +350,27 @@ func (msc *MinerSmartContract) sharderKeep(_ *transaction.Transaction,
 	buff := newSharder.Encode()
 	return string(buff), nil
 }
+
+type SharderView struct {
```

Comment:
duplicate struct

Reply:
needed for compatibility

Reply:
use existing transformer and existing structs
no duplications of struct
[minersc.minerTableToMinerNode()](https://github.com/0chain/0chain/blob/0c0e7894394e469e150e29786aeced3119107d70/code/go/0chain.net/smartcontract/storagesc/eventdb.go#L19)

Reply:
done